### PR TITLE
Cosmetic changes, mostly log-messages related.

### PR DESCRIPTION
**Pull request by OndraZizka**

### CODE REVIEW

File: bootstrap/src/main/java/org/jboss/windup/bootstrap/commands/windup/RunWindupCommand.java [hunk 1]
```diff
@@ -228,29 +233,13 @@ else if (Boolean.class.isAssignableFrom(option.getType()))
...
-            System.out.println("Input Applications:");
-            for (Path inputPath : windupConfiguration.getInputPaths())
-            {
-                System.out.println("\t" + inputPath);
-            }
-            System.out.println();
-        }
-        System.out.println("Output Path:" + windupConfiguration.getOutputDirectory());
-        System.out.println();
-
```
Comment (OndraZizka) [inline bootstrap/src/main/java/org/jboss/windup/bootstrap/commands/windup/RunWindupCommand.java]: Should this be pronted in RunWindupCommand? I think it should rather be printed in WindupProcessorImpl so it's printed in every execution path.

Comment (jsight) [inline bootstrap/src/main/java/org/jboss/windup/bootstrap/commands/windup/RunWindupCommand.java]: This goes to the user when running from the command line. WindupProcessorImp will just end up in a log file.

Comment (OndraZizka) [inline bootstrap/src/main/java/org/jboss/windup/bootstrap/commands/windup/RunWindupCommand.java]: Does the user need to see this few lines below the input command?

Comment (OndraZizka) [inline bootstrap/src/main/java/org/jboss/windup/bootstrap/commands/windup/RunWindupCommand.java]: I have left it there, no need to block this PR with that debate :)

**Comment by OndraZizka** [comment]
retest

**Comment by OndraZizka** [comment]
Also I've changed the title - this actually had not much to do with WINDUP-885.

**Pull request merged**